### ISSUE TITLE
Share inventory XML feature

### ISSUE THREAD

**Issue by btry** [flyve-mdm/android-mdm-agent#394]
I wanted to diagnose a problem with the inventory XML of my device.

I shared  the XML by email to myself. The XML contains **<?xml version='1.0' encoding='utf-8' standalone='yes' ?>** twice. It seems the XML data is written twice in the file.

```xml
<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
<REQUEST>
  <QUERY>INVENTORY</QUERY>
  <VERSIONCLIENT></VERSIONCLIENT>
  <DEVICEID>4df1f6ea5f3e7f05_38:aa:3c:16:6f:16</DEVICEID>
  <CONTENT>

(snip)

      <HEALTH>Good</HEALTH>
      <STATUS>Charging</STATUS>
    </BATTERIES>
  </CONTENT>
</REQUEST>
<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
<REQUEST>
  <QUERY>INVENTORY</QUERY>
  <VERSIONCLIENT></VERSIONCLIENT>
  <DEVICEID>4df1f6ea5f3e7f05_38:aa:3c:16:6f:16</DEVICEID>
  <CONTENT>
    <ACCESSLOG>
      <LOGDATE>2018-04-11 22:57:49</LOGDATE>

(snip)
```

**Comment by btry** [comment]
I observe the same with my sony Z2. This is very likely an error in the library or calls to it from the mdm agent project.

**Comment by rafaelje** [comment]
To prevent the append on a share file is available the version 0.7.1 related this PR https://github.com/flyve-mdm/android-inventory-library/pull/107

but to implement this library is mandatory solved this issue first https://github.com/flyve-mdm/android-mdm-agent/issues/381 and the problem with the inventory enrollment @DIOHz0r tested the XML on Friday and works fine when he imported but is not working when is send from the agent
